src/Examine.Lucene/Suggest/AnalyzingInfixSuggesterDefinition.cs

```diff
@@ -0,0 +1,93 @@
+using Lucene.Net.Analysis;
+using Lucene.Net.Index;
+using Lucene.Net.Search.Spell;
+using System.Linq;
+using Lucene.Net.Search.Suggest;
+using Lucene.Net.Search.Suggest.Analyzing;
+using Lucene.Net.Util;
+using Examine.Suggest;
+using static Lucene.Net.Search.Suggest.Lookup;
+using System.Collections.Generic;
+
+namespace Examine.Lucene.Suggest
+{
+    public class AnalyzingInfixSuggesterDefinition : LuceneSuggesterDefinition, ILookupExecutor
+    {
+        public AnalyzingInfixSuggesterDefinition(string name, string[] sourceFields = null, ISuggesterDirectoryFactory directoryFactory = null, Analyzer queryTimeAnalyzer = null)
```

Comment:
This is public and should therefore have XML documentation - Use `/// <inheritdoc />` where appropriate

Reply:
Done